stable/xray/templates/xray-statefulset.yaml

```diff
@@ -1,34 +1,32 @@
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
-  name: {{ template "xray-server.fullname" . }}
+  name: {{ template "xray.fullname" . }}
   labels:
     app: {{ template "xray.name" . }}
     chart: {{ template "xray.chart" . }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
-    component: {{ .Values.server.name }}
```

Comment:
@chukka Is there anything that still uses `.Values.server.<SOMETHING>`? I see that it is still in the `values.yaml`

Reply:
Yes  for internal and external ports - https://github.com/jfrog/charts/blob/ecd69d16b3fbdbe58e03e5399f3448e4faca1161/stable/xray/values.yaml#L541

Reply:
@chukka ok, so please remove everything in the `server` block other than those two